staging/src/k8s.io/kubectl/pkg/cmd/cmd_test.go

```diff
@@ -92,6 +136,7 @@ func TestKubectlCommandHandlesPlugins(t *testing.T) {
 			})
 
 			root := NewDefaultKubectlCommandWithArgs(pluginsHandler, test.args, in, out, errOut)
+			root.SetOut(out)
```

Comment:
This is to swallow the `kubectl help` output that would end up being printed multiple times in case of a failure of the test.